pkg/runtime/pubsub/outbox.go

```diff
@@ -21,6 +21,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/dapr/components-contrib/metadata"
+
 	"github.com/cenkalti/backoff/v4"
 	"github.com/google/uuid"
 
```

Comment:
```suggestion
"github.com/dapr/components-contrib/metadata"
```
plz fix the formatting such that this is in the same block as the other `dapr` imports